apps/dialtone-documentation/docs/.vuepress/theme/components/Navbar.vue

```diff
@@ -233,8 +334,7 @@
 
 <script setup>
 import { useRoute } from 'vue-router';
-import { onMounted, onUnmounted, inject, computed } from 'vue';
-import { setTheme } from '@dialpad/dialtone-tokens/themes/config';
+import { useThemeManager } from '../composables/useThemeManager';
```

Comment:
Love the use of composable here.
This is the kind of refactor we need to implement on Dialtone within next steps.